src/main/scala/com/microsoft/partnercatalyst/fortis/spark/Pipeline.scala

```diff
@@ -70,6 +72,11 @@ object Pipeline {
         analysis.keywords.nonEmpty
       }
 
+      def hasBlacklistedTerms(details: Details): Boolean = {
```

Comment:
Imagine if a pipeline had yet another field that they wanted to blacklist terms from.

I think `hasKeywords` and `isLanguageSupported` belong where they are in `Pipeline.scala`. We extract the keywords and language from the `Analyzer` since we need those results for our analysis. We don't care what the blacklisted terms were, so we don't need them as an output from the `Analyzer`.